docs/guide/README.md

```diff
@@ -1,94 +1,94 @@
 
 ## Building themes in Vue Storefront
 
-While making themes in Vue Storefront, in most cases, all you need to do is create your own HTML and CSS markup. All required business logic is exposed by Core with its core modules and can be easily injected into any of the theme components. The mechanism of injecting core business logic into themes is very simple. We are using Vue.js mixins to keep upgradable business logic in the core.
+While making themes in Vue Storefront in most cases all you need to take care of is creating your own HTML and CSS markup. All required business logic is exposed by the core with its core modules and can be easily injected into any of the theme components.
+The mechanism of injecting core business logic into themes is very simple. We are using Vue.js mixins to keep upgradable business logic in the core.
 
-Business logic from the core component can be easily injected into any theme component as a Vue.js mixin. We can inject it into any of our theme components just by importing and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
+Business logic from core component can be easily injected into any theme component as a Vue.js mixin. We can easily inject it into any of our theme components just by importing it and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
 
-The easiest way to create your own theme is to create a copy of the default one, change its name in its `package.json` file, change the active theme in `config/local.jso`n and run `yarn` to make lerna linking (which we are using for monorepos).
+The easiest way to create your own theme is to create a copy of the default one, change it's name in it's `package.json` file, change active theme in `config/local.json` and run `yarn` to make lerna linking (which we are using for monorepos).
 
 ## Offline mode and cache
-Vue Storefront is still working even while the user is offline.
 
-We managed to do this by making heavy use of the browser cache. 
+Vue Storefront is still working even while a user is offline.
+
+We managed to do this by making heavy use of the browser cache.
 For the static assets (only prod) we are using the sw-precache plugin (config can be found in `core/build/webpack.prod.sw.config.js` ). They are cached in Service Worker and can be inspected under `Application/Cache Storage` tab of your Developer Tools.
 
 :::warning
-Please keep in mind that Service Worker is working only on production mode.
+Please mind that Service Worker is working only on production mode.
```

Comment:
why was "keep in" from "keep in mind" removed? It made perfect sense, please change back.